haystack/nodes/label_generator/pseudo_label_generator.py

```diff
@@ -30,6 +30,24 @@ class PseudoLabelGenerator(BaseComponent):
     |   output, output_id = psg.run(documents=document_store.get_all_documents())
     |
     ```
+
+    Note:
```

Comment:
@agnieszka-m please have a quick look for clarity; we might want to make users aware of multilingual GPL in a wider context - i.e. outside of API docs (e.g. tutorials) - @julian-risch originally suggested API docs.

Reply:
I think it's a good idea to mention it in the documentation as well, I'll take care of it. 
Can you please align the parameter descriptions with what I suggested in the tutorial?